galleries/users_explain/toolkits/index.rst

```diff
@@ -0,0 +1,32 @@
+.. _tutorials-toolkits:
+
+.. redirect-from:: /tutorials/toolkits
+
+User Toolkits
+=============
+
+Here you can find examples and explanations of how to use various toolkits available in Matplotlib.
+
+Axes Artist
```

Comment:
Is there a good reason to not just have these in one TOC? 

Otherwise this looks good!

Reply:
I guess that is how I got it to look the way it did before

Reply:
Before it was a bunch of broken sphinx gallery thumbnails. https://matplotlib.org/devdocs/users/explain/toolkits/index.html  I don't think anything is gained by having a title with almost the same thing written beneath it as a one-entry table of contents.

Reply:
https://output.circle-artifacts.com/output/job/53486b53-726d-4792-9885-525ff863a886/artifacts/0/doc/build/html/users/explain/toolkits/index.html

Reply:
"The same ways as before" was just three broken thumbnails (https://matplotlib.org/devdocs/users/explain/toolkits/index.html).  Why do you want that?  

If you want something graphical, you could create an rst page with select images that are inside the tutorials. If you want the cards that are used for sphinx gallery, you could copy their style and use them on this rst page.  You could also perhaps do something with sphinx-design. 

If you decide to do this (which I'm not asking for, I'm just asking for you not to include three subheadings that each contain a table of contents) look at examples in the source code of where images have been included.